### ISSUE TITLE
数据

### ISSUE THREAD

**Issue by le010203** [Omid-Nejati/MedViTV2#9]
Why is my training result very different from yours? Did you use the validation set or test set for your data?
